scribble/app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ActiveRecord::Base
+  validates :username, presence: true, uniqueness: true
+  # validates :password_digest, presence: true
+  has_secure_password
```

Comment:
Can we think of any associations our user model can be utilized for in our application?